youtube-ios/.travis.yml

```diff
@@ -0,0 +1,14 @@
+# references:
+# * http://www.objc.io/issue-6/travis-ci.html
+# * https://github.com/supermarin/xcpretty#usage
+
+osx_image: xcode7.3
```

Comment:
there is already a .travis config file in the root directory. We can move these scripts there